src/platform/site-wide/cta-widget/components/messages/ChangeAddress.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { selectShowProfile2 } from 'applications/personalization/profile-2/selectors';
+
+import CallToActionAlert from '../CallToActionAlert';
+
+function goToOriginalProfile() {
+  window.location = '/profile/';
+}
+
+const ChangeAddress = ({
+  serviceDescription,
+  primaryButtonHandler,
+  featureToggles,
+}) => {
+  const showProfile2 = selectShowProfile2({ featureToggles });
```

Comment:
I haven't seen selectors used too often outside `mapStateToProps`, should we perhaps move this to [here](https://github.com/department-of-veterans-affairs/vets-website/pull/13804/files#diff-9d16707d98c96092904f8a12f771b0bcR494) and then pass the `showProfile2` prop to `ChangeAddress` instead? This totally works + looks great but just wanted to throw this idea out there! 🙂

Reply:
Thanks! Yeah this isn't super ideal but I did it this way because I didn't want to intermix logic specific to ChangeAddress into a shared parent component that is already getting pretty big

Reply:
Got it, I had suspected so! Looks great! 🎉